amd/src/notif.js

```diff
@@ -89,8 +81,18 @@ function fadeIn(element) {
  *
  * @param {number} feedback (given schoolgrade answer)
  * @param {text} essay (given essay answer)
- */
-const sendAndReceiveFeedback = (feedback, essay) => {
+ * @param {object} cfbParams all needed parameters:
+ *      courseId: cid,
+ *      feedbackId: fbid,
+ *      questionId: quid,
+ *      questionType: qutype,
+ *      questionSum: qusum
+ * @param {object} domElements all needed domElements:
+ *      schoolgradesContainer: Container of the schoolgrade answer elements
+ *      essayContainer: Container of the essay answer elemnts
+ *      textarea: Textarea element of the essay answer
```

Comment:
Der Vollständigkeit halber sollte hier dann auch die `notifications` property erwähnt werden (bzw. "notifi***k***ations" aktuell).

Reply:
Erledigt in: 3bb7fc84bcf96f6af9231a7b1cad7057d83c2647